### ISSUE TITLE
A list (tree?) of pages on the homepage?

### ISSUE THREAD

**Issue by mattecapu** [LocalCharts/forest#2]
Currently it's not very obvious (if at all possible) how to navigate the forest since the homepage doesn't provide any links to other pages.

Is this intentional?

**Comment by AyeGill** [comment]
You can hit CTRL-K to see a searchable list of all the pages. But this should probably be prominently displayed somewhere.

**Comment by olynch** [comment]
It is! At the bottom of every page, we have:

![image](https://github.com/LocalCharts/forest/assets/4728903/4a9dc36f-cd4f-4383-a2d9-0127f8eb9ab2)

**Comment by ralphS16** [comment]
Can we see this search window on mobile?

**Comment by mattecapu** [comment]
Yeah that utterly fails on mobile, also it's definitely not SEO friendly
Prolly a list of authors would be enough atm, and then each of us can have its own curated list in their page
